src/main/java/WayofTime/alchemicalWizardry/common/summoning/meteor/MeteorParadigm.java

```diff
@@ -115,22 +127,38 @@ public void createMeteorImpact(World world, int x, int y, int z, boolean[] flags
         }
 
         int newRadius = radius;
-
+        int fillerChance = this.fillerChance;
         if (hasOrbisTerrae) {
             newRadius += 2;
+            fillerChance *= 1.12;
         } else if (hasTerrae) {
             newRadius += 1;
+            fillerChance *= 1.6;
```

Comment:
- [ ] Are you sure these two values are not swapped? Here Terrae increases filler ratio significantly more than Orbis Terrae; which contradicts your comments and FTB wiki on the expected behavior.
- [ ] This doesn't break anything (I think), but I would still sanitize this by clamping `fillerChance` to a maximum of 100.

Reply:
> [x]  Are you sure these two values are not swapped? Here Terrae increases filler ratio significantly more than Orbis Terrae; which contradicts your comments and FTB wiki on the expected behavior. [Edit: you wrote Terrae should be 1.06, but you have 1.6 in code - that is probably the error.]

Should have been 1.06, addressed by [c8aea92](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/c8aea9243f9f4e42143b1770d4b425589e1c3e87).

> [x]  This doesn't break anything (I think), but I would still sanitize this by clamping `fillerChance` to a maximum of 100.

Addressed by [fa6cd74](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/fa6cd740536b0c0dccfdc84dd180e2f2d6df6437).